linux/src/file_util_linux.c

```diff
@@ -0,0 +1,133 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#include <string.h>
+#include <stddef.h>
+#include <stdlib.h>
+#include <fcntl.h>
+#include <inttypes.h>
+#include <stdbool.h>
+ #include <unistd.h>
+
+#include "c_pal/gballoc_hl.h" // IWYU pragma: keep
+#include "c_pal/gballoc_hl_redirect.h"
+
+#include "c_logging/logger.h"
+
+#include "c_pal/windows_defines.h"
+
+#include "c_pal/file_util.h"
+
+typedef struct CREATE_FILE_LINUX_TAG
+{
+    int h_file;
+
+} CREATE_FILE_LINUX;
+
+//Codes_SRS_FILE_UTIL_LINUX_09_002: [ file_util_create_file shall allocate memory for the file handle. file_util_create_file will succeed and return a non-NULL value. ]
```

Comment:
>file_util_create_file shall allocate memory for the file handle. file_util_create_file will succeed and return a non-NULL value. [](http://example.com/codeflow?start=38&length=128)

I have some comments on the wording of the file_util specs. I should have pointed these things out in the last review, but I missed them. Sorry for the back and forth. 

Some general comments on wording specs:

- One sentence per spec
- Format is usually: `function` shall do `action`.
- For conditional specs, the format is: If `condition`, `function` shall do action.

Reply:
The spec "`file_util_create_file` shall allocate memory for the file handle." should go before line 42.